GIT_CONVENTIONS.md

```diff
@@ -0,0 +1,87 @@
+# Git Conventions for the ORAS Community
+
+This guide is a specification for adding human and machine readable meaning to Pull Request and git commit messages in the ORAS community. It provides an easy set of rules for creating an explicit commit history; which makes it easier to explore a more structured commit history and automatically generating CHANGELOGs for each new release. 
+
+When contributing to the ORAS projects, please first discuss the change you wish to make via a GitHub issue, a GitHub discussion, or any other methods with the ORAS community maintainers before making a change.
+
+> Note: Please follow the [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) in all your interactions with the ORAS project.
+
+
+## Commit Message Guidelines
+
+This guide provides very precise rules over how Pull Requests and git commit messages.
```

Comment:
```suggestion
This guide provides very precise rules over how Pull Requests and git commit messages should be done.
```
Or you could say "should look," or something of that nature. The above (as is) isn't a complete sentence.